brainstormforce/astra
# AST-54 - fix: Customizer shortcut edit buttons has round shape whereas square shape needed

### Description
- Different edit icons on normal & Elementor sites. There should be Square edit icon everywhere in customizer.

### Screenshots
- https://share.getcloudapp.com/ApuRWbqx

### Types of changes
- Non-breaking issues

### How has this been tested?
- Check edit buttons on normal site (it looks as expected with square in shape)
- Notice the same with Elementor site (here it is round shape which is wrong)
- In this branch this should fix.

### Checklist:
